src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -54,6 +55,59 @@ public static void BasicAuthentication_RoundTrips_Echo()
     }
 
     [Fact]
+    [OuterLoop]
+    public static void BasicAuthenticationInvalidPwd_throw_MessageSecurityException()
+    {
+        StringBuilder errorBuilder = new StringBuilder();
+        //Will need to use localized string once it is available
+        string message = "The HTTP request was forbidden with client authentication scheme 'Basic'.";
```

Comment:
It turns out the exception message has stripped in .Net Native retail build. I will update it to take it into consideration.